### ISSUE TITLE
[Machine Learning] Building an image classification model with Gradio and Keras

### ISSUE THREAD

**Issue by elishanjeche** [section-engineering-education/engineering-education#7241]
#### Proposed title of article
[Machine Learning] Building an image classification model with Gradio and Keras

#### Proposed article introduction
Image classification is one of the supervised machine learning problems which aims to categorize the images of a dataset into their respective categories or labels.  Image classification is applied in computer vision to categorize an image into three or more classes.

Gradio is an open-source python library that allows you to quickly create easy-to-use, customizable UI components for your machine learning model. Gradio allows you to integrate the GUI directly into your Python notebook making it easier to use. Gradio can also be embedded in Jupyter notebooks and Google Colab notebooks. This does not require running a separate Python script. A user can interact with the model right in the working environment.

It also enables you to create demos of your machine learning model. These demos can be used to present ideas to clients, users, and team members before the actual application are implemented.

Keras is an open-source software library that provides a Python interface for artificial neural networks. Keras acts as an interface for the TensorFlow library. We will use Keras to build the image classification model and Gradio to create the model interface.

#### Key takeaways 
1. Benefits of using Gradio
2. Building a Simple Gradio interface.
3. Image preprocessing techniques in Python.
4. Initializing the Keras layers.
5. Fitting and training the image classification model.
6. Launching the Gradio interface in Google Colab.
7. Using the model to classify images of flowers and animals.
#### Article quality
This tutorial is unique because we will implement a custom image classification model from scratch.  We build an a model that classifies images of flowers and animals.  We will explain all the step of build and adding the Keras layers so that a beginner can easily follow. We will also cover all the steps involved in image preprocessing. It is a crucial step in building an image classification model. We will use `tf.keras.preprocessing` function for image classification.

After implementing the model, we will use Gradio to build a beautiful user interface. The interface will enable users to interact with the model and visualize the classification resulst. 
#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by lalith1403** [comment]
Great topic 🚀 , make sure it matches the following: 
* Adds value beyond what is in the official docs and covered in other blog sites.
* To avoid potential article closure, it should not overlap with existing EngEd articles or incoming topic suggestions.

Please reference any relevant EngEd articles in yours and build a unique project - Approved.
